src/creatures/creatures_definitions.hpp

```diff
@@ -1411,6 +1411,17 @@ struct VIPEntry {
 	bool notify;
 };
 
+struct VIPGroupEntry {
+	VIPGroupEntry(uint8_t initId, std::string initName, bool initCustomizable) :
+		id(initId),
+		name(std::move(initName)),
+		customizable(initCustomizable) { }
+
+	uint8_t id;
```

Comment:
It is recommended to initialize the primitive types, to avoid undefined behavior in the case of default constructors (in case of being added).

It might also be interesting to use string by reference instead of using std::move?

Reply:
Updated!